torch/_tensor_docs.py

```diff
@@ -2549,6 +2605,20 @@ def callable(a, b) -> number
 In-place version of :meth:`~Tensor.sqrt`
 """)
 
+add_docstr_all('square',
```

Comment:
The PR should have only added docs entries for is_quantized. Why are there other additions?

Reply:
It looks like there were some commits that removed/added some things while this one was building. I'll redo pulling in the new changes.